src/main/java/hudson/plugins/throttleconcurrents/pipeline/ThrottleStep.java

```diff
@@ -0,0 +1,81 @@
+package hudson.plugins.throttleconcurrents.pipeline;
+
+import hudson.Extension;
+import hudson.model.TaskListener;
+import hudson.plugins.throttleconcurrents.ThrottleJobProperty;
+import hudson.util.FormValidation;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.StringTokenizer;
+
+public class ThrottleStep extends Step implements Serializable {
```

Comment:
If you just have the one field, probably better to inline that into the execution state.